pkg/util/testing/wrappers.go

```diff
@@ -226,6 +226,13 @@ func (w *WorkloadWrapper) OwnerReference(apiVersion, kind, name, uid string, con
 	return w
 }
 
+// Delete sets a deletion timestamp for the pod object
+func (w *WorkloadWrapper) Delete() *WorkloadWrapper {
+	t := metav1.NewTime(time.Now()).Rfc3339Copy()
+	w.Workload.DeletionTimestamp = &t
```

Comment:
```suggestion
	w.Workload.DeletionTimestamp := ptr.To(metav1.NewTime(time.Now()).Rfc3339Copy())
```

Reply:
> What is this `Rfc3339Copy` doing? truncating to second?

Correct, had to add it because of https://github.com/kubernetes/kubernetes/issues/81026